diff -ad -uNrp a/lib/lib.raumserver.js b/lib/lib.raumserver.js
--- a/node-raumserver/lib/lib.raumserver.js	2019-12-22 01:10:55.000000000 +0100
+++ b/node-raumserver/lib/lib.raumserver.js	2019-12-22 01:11:36.000000000 +0100
@@ -42,7 +42,7 @@ module.exports = class Raumserver extend
         var self = this;
 
         // set the template file for uri set method
-        this.raumkernel.getSettings().uriMetaDataTemplateFile =  "node_modules/node-raumkernel/lib/setUriMetadata.template";
+        this.raumkernel.getSettings().uriMetaDataTemplateFile =  "/node_modules/node-raumkernel/lib/setUriMetadata.template";
 
         // set some other settings from the config/settings file
         // TODO: @@@
